src/adapter/src/coord/sequencer/old_optimizer_api.rs

```diff
@@ -0,0 +1,347 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! This module contains alternative implementations of `sequence_~` methods
+//! that optimzie plans using the new optimizer API. The `sequence_plan` method
+//! in the parent module will delegate to these methods only if the
+//! `enable_unified_optimizer_api` feature flag is on. Once we have gained
```

Comment:
```suggestion
//! `enable_unified_optimizer_api` feature flag is off. Once we have gained
```

... right?

Reply:
Good catch! Originally this file contained the new implementations, but I swapped this in this PR in order to minimized the diff in the follow-up PR that will remove these methods.